react/actions/order-actions.js

```diff
@@ -39,9 +39,11 @@ const ORDERS_DETAILED_LOADED_LIMIT = 3
 let listingResponseGuid = ''
 
 async function loadOrders({ page, guid }) {
+  const { account } = window.__RUNTIME__
```

Comment:
How does this work? I didn't see this (`window.__RUNTIME__`) being used anywhere else in vtex-apps.

Reply:
This `window.__RUNTIME__` variable is injected automatically by the render framework into all frontend pages. It's not meant to be used directly in frontend code, we have  the `vtex.render-runtime` package for this instead. Unfortunately, `vtex.render-runtime` only provides this data via a hook, which can only be used directly within function components. I've taken a quick look here and unfortunately it would take **A LOT** of effort to adapt this code to use `vtex.render-runtime` 🤦‍♀️ , so I don't think we have a better option here... 😭

Reply:
Thanks @mairatma for the context. I found this solution by searching for the code on Github and in these [docs](https://tech-docs.vtex.com/Storefronts/Store-Framework/general-debugging-questions/?h=__runtime#is-the-information-from-window__runtime__-dynamic).